packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/compute/DatacentreCompute.page.tsx

```diff
@@ -49,6 +54,9 @@ export default function ComputeListingPage() {
   const { t } = useTranslation('hpc-vmware-managed-vcd/datacentres/compute');
   const { t: tVdc } = useTranslation('hpc-vmware-managed-vcd/datacentres');
   const navigate = useNavigate();
+  const { data: vcdOrganization, isLoading } = useManagedVcdOrganization({
```

Comment:
error management is missing

Reply:
I removed this part since we don't need to check for IAM rights here anymore